src/Components/FormContainer/FormContainer.tsx

```diff
@@ -29,6 +29,11 @@ export const getQuestionsQuery = gql`
 `;
 
 export const FormContainer = (props: FormContaineProps) => {
+  let id = props.userId
```

Comment:
Awesome job error handling here incase the user refreshes the page, that makes for a much better UX